Content.Shared/Radio/Components/HeadsetComponent.cs

```diff
@@ -15,4 +17,16 @@ public sealed partial class HeadsetComponent : Component
 
     [DataField("requiredSlot")]
     public SlotFlags RequiredSlot = SlotFlags.EARS;
+
+    /// <summary>
+    /// Sound of receiving message from toggled channel.
+    /// </summary>
+    [DataField]
+    public SoundSpecifier Sound;
+
+    /// <summary>
+    /// Headset will emit sound when receives message from these channels.
+    /// </summary>
+    [DataField("soundChannels")]
+    public HashSet<ProtoId<RadioChannelPrototype>> ToggledSoundChannels = [];
```

Comment:
Unless there is a good reason, probably don't rename the field unless really necessary. It gets very confusing when the names are mismatched! Just remove the override name from the datafield and change they yaml